.github/workflows/codeql.yml

```diff
@@ -0,0 +1,77 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
Can we include a link to the original copy?
I assume we copied this file from some other places.

Reply:
It is auto generated by GitHub. (Settings -> Code security and analysis -> CodeQL analysis -> Set up -> Advanced)

Reply:
I see.
Then we are good.